test/unit/lib/is-array.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
I meant the version of the isArray function that uses `toString` to determine array-ness.